lit/docs/operation/tracing.lit

```diff
@@ -34,6 +34,15 @@ although support for other systems is planned to expand as the underlying SDK
   \link{\code{GOOGLE_APPLICATION_CREDENTIALS} environment variable}{https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable},
   the default location that the \code{gcloud} CLI expects, or from GCP's
   metadata server (if Concourse is deployed on GCP).
+  
```

Comment:
I think the build failure is due to whitespaces on empty lines, I was able to build it locally after removing the 2 spaces on this line.

Reply:
I'll fix this up and the other suggestion. Thanks for the review!